module/VuFind/src/VuFind/Db/Table/Comments.php

```diff
@@ -77,7 +77,8 @@ public function getForResource($id, $source = DEFAULT_SEARCH_BACKEND)
             $select->columns(['*']);
             $select->join(
                 ['u' => 'user'], 'u.id = comments.user_id',
-                ['firstname', 'lastname']
+                ['firstname', 'lastname'],
+                'left'
```

Comment:
We should use the `$select::JOIN_LEFT` constant here in place of the string `'left'`.